modules/01 JavaScript Programming/Assignments/19/answerJ19.txt

```diff
@@ -0,0 +1,125 @@
+
+var errorsObjects={
+    valid:true,
+    size:{
+        valid:true,
+        error:'wrong_length',
+        isSizeValide: function(array,size)
```

Comment:
Placing a function as an object value is dangerous because it can easily get overwritten. A private function or a constructor would be safer.